src/features/commands/watch.js

```diff
@@ -0,0 +1,38 @@
+const { SlashCommandBuilder } = require('@discordjs/builders'),
+      db = require('../../db');
+
+module.exports = {
+    type: 'slash',
+    data: new SlashCommandBuilder()
+        .setName('watch')
+        .setDescription('Sends a direct message whenever keyword that you track gets mentioned.')
+        .addStringOption(option =>
+            option.setName('keyword')
+                .setDescription('The keyword you track.')
+                .setRequired(true)
+        ),
+    async execute (client, interaction) {
+        const keyword = interaction.options.getString("keyword");
```

Comment:
```suggestion
        const keyword = interaction.options.getString("keyword", true);
```
Adding true to option getters like getString means it will throw an error if keyword is nullish for whatever reason.